app/assets/javascripts/discourse/components/user-stream.js.es6

```diff
@@ -8,6 +8,16 @@ import { popupAjaxError } from "discourse/lib/ajax-error";
 import { getOwner } from "discourse-common/lib/get-owner";
 
 export default Ember.Component.extend(LoadMore, {
+  _initialize: function() {
+    const stream = this.get("stream");
```

Comment:
Can you do this instead?

```suggestion
    const filter = this.get("stream.filter");
```

Reply:
Oh, didn't know that was possible. Thanks! ❤️ Any idea on the failing tests though? Been on it for some hours now and still can't get them running correctly.

Reply:
cc @ZogStriP finally managed to fix them :) Also added a test